project_name/settings.py

```diff
@@ -11,20 +11,30 @@
 """
 
 import os
+from os.path import dirname, join, exists, abspath
+
+import environ  
+
+# Load operating system env variables and prepare to use them
+env = environ.Env()
+
+# .env file, should load only in development environment
+env_file = join(dirname(__file__), 'local.env')
```

Comment:
Does it just get the vars from `local.env`? I think people are more used to just `.env`.